cmd/kube-apiserver/app/server.go

```diff
@@ -491,13 +492,16 @@ func BuildAuthorizer(s *options.ServerRunOptions, EgressSelector *egressselector
 }
 
 // BuildPriorityAndFairness constructs the guts of the API Priority and Fairness filter
-func BuildPriorityAndFairness(s *options.ServerRunOptions, extclient clientgoclientset.Interface, versionedInformer clientgoinformers.SharedInformerFactory) utilflowcontrol.Interface {
+func BuildPriorityAndFairness(s *options.ServerRunOptions, extclient clientgoclientset.Interface, versionedInformer clientgoinformers.SharedInformerFactory) (utilflowcontrol.Interface, error) {
+	if s.GenericServerRunOptions.MaxRequestsInFlight+s.GenericServerRunOptions.MaxMutatingRequestsInFlight <= 0 {
```

Comment:
Personally I'd check each individually. Otherwise one of them could be zero or less.

Reply:
I assumed there is validation on each individually

Reply:
Yes, that's in `staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go`

Reply:
Since we just add them together before constructing the filter anyway, the changes in this file seems fine. But can you also update the flag documentation at the bottom of `staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go` to explain that 0 no longer has the unlimited meaning when APF is on?

Reply:
The force-push to 438942acf7f adds an update to the max inflight flag help at the end of `staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go`.